### PR TITLE
adding new ideas by registered & logged users

### PR DESCRIPTION
**Pull request by moraw9**
# tasks
- [x] create layout for button
- [x] create form for users
- [x] validate input data 
- [x] save data on firestore 
- [x] reload data displayed on the page
- [x] close the form after saving and clear if close without saving data
- [x] clear changeset after saving data
- [ ] change modal form structure - add new component: field, after get it from another branch

### CODE REVIEW

**Comment by wozny1989** [comment]
@moraw9  Add space between `#` nad `task` i PR description - it should display much better (like header) 😄

**Comment by wozny1989** [comment]
Nice work with create list with tasks! 🚀

File: app/components/adding-new-idea/template.hbs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="modal-header">
+          <h4 class="modal-title">Enter your project information</h4>
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+        </div>
+        
+        <div class="modal-body">
+          <form>
+            <div class="form-group">
+              <label for="name">Project title:</label>
+              <input type="text" class="form-control" id="title" placeholder="Enter project title" name="title" value={{this.changeset.title}} onchange={{fn this.setValue}} >
```
Comment (wozny1989) [inline app/components/adding-new-idea/template.hbs]: I'm not sure is `fn` helper is necessary when we call function `this.setValue` without properties. Try do this, and give me a feedback if it still work:
```suggestion
              <input type="text" class="form-control" id="title" placeholder="Enter project title" name="title" value={{this.changeset.title}} onchange={{this.setValue}} >
```

Comment (moraw9) [reply app/components/adding-new-idea/template.hbs]: Yes, it works without 'fn', I corrected it.

Comment (wozny1989) [reply app/components/adding-new-idea/template.hbs]: Cool!

File: app/helpers/getExcerpt.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { helper } from '@ember/component/helper';
 
 function getExcerpt([string]) {
+  // console.log('string', string);
```
Comment (wozny1989) [inline app/helpers/getExcerpt.js]: ✂️

**Review by wozny1989 — CHANGES_REQUESTED** [review]

**Pull request merged**